projects/igniteui-angular/migrations/update-12_1_0/changes/members.json

```diff
@@ -246,6 +246,13 @@
                 "IgxSelectComponent"
             ]
         },
+        {
+            "member": "onSelection",
+            "replaceWith": "selecting",
```

Comment:
The event in the igx-combo is renamed to selectionChange. Why it is different in the igx-drop-down and igx-select, @valadzhov?